tests/ttnn/unit_tests/operations/test_maxpool2d.py

```diff
@@ -107,6 +107,7 @@ def run_max_pool(
         stride=[stride_h, stride_w],
         padding=[pad_h, pad_w],
         dilation=[dilation_h, dilation_w],
+        # memory_config=ttnn.DRAM_MEMORY_CONFIG,
```

Comment:
Please don't add commented out code.

If you want to test a specific memory config add code for it. If you want to say that a specific config will be used just add a comment stating that.